data/hypha.html

```diff
@@ -7,6 +7,7 @@
     <div id="navContainer" class= "container navContainer">
       <nav class="nav main">
         <div id="menu" class="nav-left"/>
+        <div id="review" class="nav-left"/>

```

Comment:
This is a nice feature, although the logic for `a list of articles that need a review` should be in the `peer_reviewed_article`.